Makefile

```diff
@@ -67,7 +67,12 @@ git-blame:
 .git/hooks/pre-commit: scripts/pre-commit
 	./scripts/setup-pre-commit-hook
 
-dev-env: $(PELORUS_VENV) exporters git-blame .git/hooks/pre-commit
+$(PELORUS_VENV)/binary_tools_installed:
+	./scripts/install_dev_tools -v $(PELORUS_VENV)
+	touch $(PELORUS_VENV)/binary_tools_installed
```

Comment:
In my last comment, I realized the old way you were doing it was fine. This way has the downside you pointed out where deleting one won't trigger an update.

Reply:
@KevinMGranger ok changed back. Thanks.